docs/ReproducibilityChecklists-DataModels.md

```diff
@@ -0,0 +1,80 @@
+| `answer`           | String     | Stores the user's response to the checklist item. ('yes'/'no' for boolean type, 'answer description' for descriptive type)                                                                                                         |
+| `userNotes`        | Array ([]Note)| An array containing user-added notes associated with the checklist item.                                                                                  |
+| `attachedImages`   | Array ([]Image)| An array containing image data of images attached to the checklist item.                                                                                  |
+| `attachedURLs`     | Array ([]URL)| An array containing URLs attached to the checklist item.                                                                                     |
+| `subChecklists`    | Array ([]Sub_checklist)| An array containing sub-checklists associated with the checklist item.                                                                                    |
+| `updated`          | String     | The timestamp indicating when the checklist item was last updated.                                                                                         |
+
+### Sub-Checklist Object
+
+#### About
+
+The **Sub-Checklist Object** represents a sub-item within a checklist item. It shares similar attributes with the Checklist Object, except for attachments, but is nested within the parent checklist item.
+
+#### Attributes
+
+| Attribute       | Type     | Description                                                                                                                                               |
+| --------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`            | UUID     | A generated unique identifier for the sub-checklist item.                                                                                                           |
+| `statement`     | String   | The statement or question associated with the sub-checklist item.                                                                                          |
+| `type`          | String   | Indicates the type of sub-checklist item, such as boolean or descriptive.                                                                                  |
+| `answer`        | String   | Stores the user's response to the sub-checklist item. ('yes'/'no' for boolean type, 'answer description' for descriptive type)                                                                                         |
+| `updated`       | String   | The timestamp indicating when the sub-checklist item was last updated.                                                                                     |
+
+### Note Object
```

Comment:
Totally fine to have this documented here, but maybe add a comment that says "This follows the exact same structure as: https://github.com/StatTag/StatWrap/blob/master/docs/Notes.md", just so we know that it's meant to be the same (and a reminder to ourselves to update documentation in both locations, if needed)

Reply:
sure! i remember you calling that out earlier as well for the proposal 😅